test/e2e/specs/editor/various/block-renaming.spec.js

```diff
@@ -79,7 +79,11 @@ test.describe( 'Block Renaming', () => {
 
 			await expect( saveButton ).toBeEnabled();
 
-			await saveButton.click();
+			await page.keyboard.press( 'Tab' );
+			await page.keyboard.press( 'Tab' );
+
+			await expect( saveButton ).toBeFocused();
+			await page.keyboard.press( 'Enter' );
```

Comment:
This shouldn't be necessary to change. Something about `await saveButton.click();` was throwing in a wrench in it. If I try the previous test manually it works. I debugged for awhile but couldn't isolate the reason, so moved on using this test.

I have a concern that it's failing for a valid reason, but since I can't reproduce when I do the test manually, I think it's OK.